scania-digital-design-system/sdds
# feat(side menu): sticky collapse button and popover example fix

**Describe pull-request**  
- placed the side menu collapse button  above the other items in the side menu and fixed on the bottom when you scroll the page
- side-menu pop-over example is fixed accordingly 

**Solving issue**  
Fixes: [AB#1164](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1164) [AB#985](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/985)
Fixes: #190 

**How to test**  

1. Go to storybook side menu example 
2. Check the sticky collapse button and pop-over example


**Screenshots**  
_If applicable, add screenshots to help explain_

**Additional context**  
_Add any other context about the pull-request here._
